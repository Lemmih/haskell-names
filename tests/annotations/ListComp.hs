module ListComp where

a b = [ (x,y,z,t)
      | x <- b
      , y <- x
      , let t = y
      , y <- y
      , t
      , x
      , z <- x
      ]
