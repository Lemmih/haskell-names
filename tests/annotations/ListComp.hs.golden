x        at 3:10 is a local value defined at 4:9
y        at 3:12 is a local value defined at 7:9
z        at 3:14 is a local value defined at 10:9
t        at 3:16 is a local value defined at 6:13
b        at 4:14 is a local value defined at 3:3
x        at 5:14 is a local value defined at 4:9
y        at 6:17 is a local value defined at 5:9
y        at 7:14 is a local value defined at 5:9
t        at  8:9 is a local value defined at 6:13
x        at  9:9 is a local value defined at 4:9
x        at 10:14 is a local value defined at 4:9
